packages/contracts/src/namespaces/root/systems/AllowanceSystem.sol

```diff
@@ -0,0 +1,167 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.24;
+
+import { System } from "@latticexyz/world/src/System.sol";
+
+import { Balance } from "../codegen/tables/Balance.sol";
+import { Allowance, AllowanceData } from "../codegen/tables/Allowance.sol";
+import { AllowanceList, AllowanceListData } from "../codegen/tables/AllowanceList.sol";
+import { BlockedAllowance } from "../codegen/tables/BlockedAllowance.sol";
+import { SystemConfig } from "../codegen/tables/SystemConfig.sol";
+import { IEntryPoint } from "@account-abstraction/contracts/interfaces/IEntryPoint.sol";
+
+uint256 constant MIN_ALLOWANCE = 10_000;
```

Comment:
do we want to denote this in gwei or ether?

Reply:
yeah this value is also kind of arbitrary rn, just used 10k because it was used on some test, but we can increase it and denote in eth